src/components/SimpleSlider.js

```diff
@@ -0,0 +1,173 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+const _merge = require('lodash/merge');
+
+const StyleConstants = require('../constants/Style');
+
+const SimpleSlider = React.createClass({
+  propTypes: {
+    disabled: React.PropTypes.bool,
+    onPercentChange: React.PropTypes.func.isRequired,
+    percent: React.PropTypes.number.isRequired,
+    selectedColor: React.PropTypes.string,
+    styles: React.PropTypes.object
+  },
+
+  getDefaultProps () {
+    return {
+      disabled: false,
+      selectedColor: StyleConstants.Colors.PRIMARY
+    };
+  },
+
+  getInitialState () {
+    return {
+      dragging: false,
+      leftPixels: 0,
+      width: 0
+    };
+  },
+
+  componentDidMount () {
+    this._setDefaults();
+  },
+
+  componentWillReceiveProps (newProps) {
+    if (this.props.percent !== newProps.percent) {
+      const leftPixels = newProps.percent * this.state.width;
+
+      this.setState({ leftPixels });
+    }
+  },
+
+  _setDefaults () {
+    const component = ReactDOM.findDOMNode(this.rangeSelectorRef);
+    const width = component.clientWidth;
+    const leftPixels = this.props.percent * width;
+
+    this.setState({ width, leftPixels });
+  },
+
+  _handleMouseEvents (e) {
+    const clientX = e.touches ? e.touches[0].clientX : e.clientX;
+    const leftSpace = ReactDOM.findDOMNode(this.rangeSelectorRef).getBoundingClientRect().left;
+    let currentPercent = ((clientX - leftSpace) / this.state.width);
```

Comment:
No need for outer curly braces
`let currentPercent = (clientX - leftSpace) / this.state.width;`